samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/author/ExportPoolListener.java

```diff
@@ -0,0 +1,101 @@
+/**********************************************************************************
+ * $URL$
+ * $Id$
+ ***********************************************************************************
+ *
```

Comment:
Update the header!

https://github.com/sakaiproject/sakai/blob/master/ECLv2-header-template.txt